src/ray/raylet/node_manager.cc

```diff
@@ -1472,7 +1472,10 @@ void NodeManager::ProcessReportActiveObjectIDs(
   std::shared_ptr<Worker> worker = worker_pool_.GetRegisteredWorker(client);
   if (!worker) {
     worker = worker_pool_.GetRegisteredDriver(client);
-    RAY_CHECK(worker);
+    if (!worker) {
+      RAY_LOG(ERROR) << "Ignoring object ids report from failed / unknown worker.";
+      return;
+    }
```

Comment:
Is the report active ID still used? Thought it has been disabled.

Reply:
Disabled by default but still can be used by setting config option. Plan is to keep it that way in case people have issues with eviction before GC is merged.